src/svg/cssEmphasis.ts

```diff
@@ -0,0 +1,82 @@
+import LRU from '../core/LRU';
+import { extend, isGradientObject, isString, map } from '../core/util';
+import * as colorTool from '../tool/color';
+import Displayable from '../graphic/Displayable';
+import { GradientObject } from '../graphic/Gradient';
+import { BrushScope, SVGVNodeAttrs } from './core';
+
+// TODO: Consider deleting the same logic in ECharts and call this method?
+const liftedColorCache = new LRU<string>(100);
+function liftColor(color: GradientObject): GradientObject;
+function liftColor(color: string): string;
+function liftColor(color: string | GradientObject): string | GradientObject {
+    if (isString(color)) {
+        let liftedColor = liftedColorCache.get(color);
+        if (!liftedColor) {
+            liftedColor = colorTool.lift(color, -0.1);
+            liftedColorCache.put(color, liftedColor);
+        }
+        return liftedColor;
+    }
+    else if (isGradientObject(color)) {
+        const ret = extend({}, color) as GradientObject;
+        ret.colorStops = map(color.colorStops, stop => ({
+            offset: stop.offset,
+            color: colorTool.lift(stop.color, -0.1)
+        }));
+        return ret;
+    }
+    // Change nothing.
+    return color;
+}
+
+export function createCSSEmphasis(
+    el: Displayable,
+    attrs: SVGVNodeAttrs,
+    scope: BrushScope
+) {
+    if (!el.ignore && el.__metaData) {
+        const empahsisStyle = el.states.emphasis
+            ? el.states.emphasis.style
+            : {};
```

Comment:
It seems the [`emphasis.disabled`](https://echarts.apache.org/zh/option.html#series-bar.emphasis.disabled) option is not being respected. The chart still highlights the item that the emphasis state is disabled.